src/DiagnosticMessages.ts

```diff
@@ -1054,6 +1055,11 @@ export let DiagnosticMessages = {
         legacyCode: 1154,
         severity: DiagnosticSeverity.Error,
         code: 'cannot-find-callfunc'
+    }),
+    notCallable: (name: string) => ({
```

Comment:
Added new Diagnostic for when you're trying to call on something that isn't a function